command/agent/agent_endpoint.go

```diff
@@ -333,6 +334,92 @@ func (s *HTTPServer) AgentForceLeaveRequest(resp http.ResponseWriter, req *http.
 	return nil, err
 }
 
+func (s *HTTPServer) AgentPprofRequest(resp http.ResponseWriter, req *http.Request) ([]byte, error) {
+	path := strings.TrimPrefix(req.URL.Path, "/v1/agent/pprof/")
+	switch path {
+	case "":
+		// no root index route
+		return nil, CodedError(404, ErrInvalidMethod)
```

Comment:
Having an index page would be super nice

Reply:
Because of its multi-node functionality, I'm not sure what the index page would display? think having a nomad-ui interface would be great. I'm happy to add something though, do you have any thoughts on what it might respond with?